vnc_auto.html

```diff
@@ -81,11 +44,19 @@
 
         let rfb;
         let desktopName;
+        let screen_div;
+        let screen_canvas;
 
         // When this function is called we have
         // successfully connected to a server
         function connectedToServer(e) {
             status("Connected to " + desktopName);
+            screen_div = rfb._screen;
+            screen_div.id = "canvasCtnr";
+            screen_canvas = rfb._screen.firstChild
+            screen_canvas.id = 'canvasArea';
+            screen_canvas.style.margin = null;
+            rfb.focus();
```

Comment:
Move into our own js `idrive_vnc_base.js` and include in body.

Reply:
@diganta-dutta CSS related changes have been removed but `rfb.focus();` function is getting called in `connect` event. I have tried putting the same in custom JS file but things didn't work as expected
- I tried to get the DOM elements but it seems RFB object construction and then connect function takes time and the DOM is not ready yet so I am not able to set the focus
- I tried other ways too but didn't help
Overall, I would say for now, lets keep this change here only

Reply:
Ok